examples/minicpmv_example.py

```diff
@@ -0,0 +1,39 @@
+from PIL import Image
+
+from vllm import LLM, SamplingParams
+
+IMAGES = [
+    "./example.png",
+]
+
+MODEL_NAME = "openbmb/MiniCPM-V-2"
+
+image = Image.open(IMAGES[0]).convert("RGB")
```

Comment:
Please note that images from `s3://air-example-data-2/vllm_opensource_llava/` can now be downloaded automatically when accessed via `vllm.assets.image.ImageAsset`.